tests/__snapshots__/app.test.ts.snap

```diff
@@ -552,6 +552,8 @@ declare global {
 exports[`integrates together: nexus prisma typegen 1`] = `
 "import * as photon from '@prisma/photon';
 import { core } from 'nexus';
+import { GraphQLResolveInfo } from 'graphql';
```

Comment:
I think `graphql` can remain a dev dep since we are just importing types from it.

This comment can be addressed in a follow up commit.

Reply:
If someone consuming nexus-prisma didn't have ```graphql``` installed but was using TS, how would ```import { GraphQLResolveInfo } from 'graphql';``` be resolved in the d.ts file? Also ```nexus``` already has ```graphql``` as a peerDependency so effectively ```nexus-prisma``` does as well.